samples/msal-node-samples/on-behal-0f/web-app/index.js

```diff
@@ -0,0 +1,80 @@
+
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+const express = require("express");
+const msal = require('@azure/msal-node');
+const http = require('http')
+
+const SERVER_PORT = process.env.PORT || 3000;
+
+// Before running the sample, you will need to replace the values in the config, 
+// including the clientSecret
+const config = {
+    auth: {
+        clientId: "12d77c73-d09d-406a-ae0d-3d4e576f7d9b",
+        authority: "https://login.microsoftonline.com/90b8faa8-cc95-460e-a618-ee770bee1759",
+        clientSecret: "",
```

Comment:
I would use the `dotenv` package with `.env` file to have the secret loaded via environment variables.

Reply:
https://www.npmjs.com/package/dotenv

Reply:
This is probably something we should do across the sample framework (@technical-boy)

Reply:
Since all the samples are in the process of being rewritten, I'll let @technical-boy decide how to handle storing secrets in a way that works for all of the samples/tests.